### PR TITLE
bpo-41260: C impl of datetime.date.strftime() takes different keyword arg

### PR DESCRIPTION
**Pull request by ZackerySpytz**
The parameter was named "fmt" in the Python implementation, but it
was was named "format" in the C implementation.

https://bugs.python.org/issue41260

### CODE REVIEW

**Review by asottile — COMMENTED** [review]
oh uh, I was [working through this with a beginner as a mentorship opportunity](https://bugs.python.org/msg373424) -- but the patch looks correct and equivalent to what we were doing

there's another function which should be updated as well which has a similar problem

**Review by pganssle — APPROVED** [review]

File: Lib/test/datetimetester.py [hunk 1]
```diff
@@ -1489,6 +1489,9 @@ def test_strftime(self):
         #check that this standard extension works
         t.strftime("%f")
 
+        # bpo-41260: The parameter was named "fmt" in the pure python impl.
```
Comment (iritkatriel) [inline Lib/test/datetimetester.py:1492]:
```suggestion
        # bpo-41260: The parameter was named "fmt" in the pure python impl.
```

**Comment by iritkatriel** [comment]
Should this be backported?

**Comment by asottile** [comment]
> Should this be backported?

I don't think so -- since it ~technically changes the signatures of the functions slightly

**Pull request merged**